### PR TITLE
Taxjar fix

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> Productized fix for TaxJar's responsiveness bugs

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

 - Jira issue number: https://saasquatch.atlassian.net/browse/CT-196
 - Process.st launch checklist: https://app.process.st/checklists/TaxJar-bug-fix-hZ3rwsSRRMd7k00lhNRIhg

### Development

- [x] [Prettier](https://www.npmjs.com/package/prettier) was run
- [x] The behaviour changes in the pull request are covered by specs
- [x] All tests related to the changed code pass in development

### Paperwork

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has a Jira number
- [x] This pull request has a Process.st launch checklist
- [x] "Ready for review" label attached to the PR and reviewers pinged on Slack

### Code review 

- [x] Changes have been reviewed by at least one other engineer
- [x] Security impacts of this change have been considered

### CODE REVIEW

File: templates/widget.hbs [hunk 1]
```diff
@@ -36,6 +36,11 @@
       body .squatch-container {
         font-family: {{variables 'widget.REFERRER_WIDGET.content.widgetCustomization.widgetFont'}};
       }
+      .squatch-container-popup {
+        height: {{variables 'widget.REFERRER_WIDGET.content.widgetCustomization.popupHeight'}};
+        margin-top: {{variables 'widget.REFERRER_WIDGET.content.widgetCustomization.popupMarginTop'}};
```
Comment (jorgecgll) [inline templates/widget.hbs]:
```suggestion
        margin: {{variables 'widget.REFERRER_WIDGET.content.widgetCustomization.popupMarginTop'}};
```

File: variables.json [hunk 2]
```diff
@@ -174,6 +174,21 @@
...
+                                       "title":"Popup Height",
+                                       "type":"string",
+                                       "default":"486px"
+                                    },
+                                    "popupZIndex":{
+                                       "title":"Popup Z Index",
+                                       "type":"integer",
+                                       "default":1
+                                    },
+                                    "popupMarginTop":{
```
Comment (jorgecgll) [inline variables.json]: This has to be something more flexible. Some people might want to add other margins

**Review by jorgecgll — CHANGES_REQUESTED** [review]

File: variables.json (hunk 2 above)
Comment (00salmon) [reply variables.json]: sounds good 👍

Comment (00salmon) [reply variables.json]: done!

**Review by jorgecgll — APPROVED** [review]
looks good

**Pull request merged**